arangod/Pregel/Graph.h

```diff
@@ -48,6 +48,10 @@ struct PregelID {
   bool operator==(const PregelID& rhs) const {
     return shard == rhs.shard && key == rhs.key;
   }
+  
+  bool operator!=(const PregelID& rhs) const {
+    return shard != rhs.shard || key != rhs.key;
```

Comment:
this is normally implemented as the negation of the `==` operator, but it is ok as it is:
```suggestion
    return !(*this == rhs);
```